django_project/api/osm_api_client.py

```diff
@@ -193,6 +214,8 @@ def update_node(self, data, comment=None):
                 'visible': True|False
             }
         """
+        current_data = self.NodeGet(data['id'])
```

Comment:
@meomancer Just a question: `NodeGet` and `WayGet` are now pulling the latest version from OSM before update. Is this correct?

Reply:
yes @nixilla 
NodeGet and WayGet will get data from latest version from OSM
After that we merge the data, so we we don't delete anything from osm
the test:
https://github.com/healthsites/healthsites/pull/1359/files#diff-b618f14721bb1399f8e99f3fd7fc3e02R160